### ISSUE TITLE
 How can I add a configuration when running the coverage?

### ISSUE THREAD

**Issue by grupo-exito-ecommerce** [vtex/test-tools#9]
We need to run the tests with the following configuration

`coverageThreshold: {
            "global": {
              "branches": 80,
              "functions": 80,
              "lines": 80
}`

**Comment by Maik3345** [comment]
?

**Comment by klzns** [comment]
There's no way now. 

Maybe you could open a PR doing something similar like we do with the option of `setupFilesAfterEnv`:
https://github.com/vtex/test-tools/blob/master/modules/createJestConfig.js#L11-L13

So if you have in your `package.json`:

```jsonc
{
  // ...
  "jest": {
    "coverageThreshold": {
      "global": {
        "branches": 80,
        "functions": 80,
        "lines": 80,
        "statements": -10
      }
    }
  }
}
```

We would read it and pass it down to jest.

**Comment by lucasecdb** [comment]
closing as it was fixed in #31
